### PR TITLE
Dispatcher: translate QuickMenu title

### PR DESCRIPTION
**Pull request by hius07**
Closes https://github.com/koreader/koreader/issues/10270.

- - -
This change is [Reviewable](https://reviewable.io/reviews/koreader/koreader/10271)

### CODE REVIEW

File: frontend/dispatcher.lua [hunk 1]
```diff
@@ -967,7 +967,7 @@ function Dispatcher:_showAsMenu(settings)
     end
     local ButtonDialogTitle = require("ui/widget/buttondialogtitle")
     quickmenu = ButtonDialogTitle:new{
-        title = settings.settings.name or "Quick Menu",
+        title = settings.settings.name or _("QuickMenu"),
```
Comment (Frenzie) [inline frontend/dispatcher.lua:970]: Is the space removal on purpose?

**Comment by Frenzie** [comment]
Ftr you can immediately squash & merge such minor obvious fixes, although in this case I'm glad you didn't due to the slightly changed string. ;-)

File: frontend/dispatcher.lua (hunk 1 above)
Comment (Frenzie) [reply frontend/dispatcher.lua:970]: Given our usual style I think it should probably be changed as follows:
```suggestion
        title = settings.settings.name or _("Quick menu"),
```

**Comment by hius07** [comment]
Withous a space everywhere:
https://github.com/koreader/koreader/blob/3a37d30ad6a98da672d31921bc37c300607f2c74/frontend/dispatcher.lua#L905
https://github.com/koreader/koreader/blob/3a37d30ad6a98da672d31921bc37c300607f2c74/plugins/profiles.koplugin/main.lua#L134

**Pull request merged**